mleap-core/src/main/scala/ml/combust/mleap/core/feature/MathUnaryModel.scala

```diff
@@ -56,3 +61,28 @@ case class MathUnaryModel(operation: UnaryOperation) extends Model {
   override def outputSchema: StructType = StructType(
     "output" -> ScalarType.Double.nonNullable).get
 }
+
+object LogitHelper { 
+  val CLIPPING_MIN = 0.000001
+  val CLIPPING_MAX = 0.999999
+  val LOGIT_MIN = 0.0
+  val LOGIT_MAX = 1.0
+  
+  def clipToLogitRange(value: Double) : Double = { 
```

Comment:
Why add this clipping logic? I think it would be better to just let it raise `OutOfRangeException`
Then users can manually clip if they want to circumvent this

Reply:
Removed